Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -994,6 +1059,13 @@ - (void)setSelectionState:(NSDictionary *)selectionDetails
 		[separatorTableMenuItem3 setHidden:NO];
 		[openTableInNewTabMenuItem setTitle:NSLocalizedString(@"Open View in New Tab", @"open view in new table title")];
 		[openTableInNewWindowMenuItem setTitle:NSLocalizedString(@"Open View in New Window", @"Tables List : Gear Menu : Duplicate connection to new window")];
+		[pinTableMenuItem setHidden:NO];
+        if (isCurrentSelectionPinned) {
+            [pinTableMenuItem setTitle:NSLocalizedString(@"Unpin View", @"pin view menu item title")];
+        }
+        else {
+            [pinTableMenuItem setTitle:NSLocalizedString(@"Pin View", @"pin view menu item title")];
+        }
```

Comment:
For all of these text spots, it looks like you're getting the text twice (like the if block is exactly duplicated)? Maybe you should get it once and store it in a variable or abstract it to a function?